src/Atmos/Dycore/src/VanillaAtmosDiscretizations_cuda.jl

```diff
@@ -0,0 +1,1242 @@
+# FIXME: Add link to https://github.com/paranumal/libparanumal here and in
```

Comment:
I'm assuming you didn't mean to add this file?

Reply:
Yes - this is unintentional